tos.html

```diff
@@ -8,17 +8,17 @@ <h2 id="1-who-can-use-the-service">1. Who can use the Service</h2>
 </ul>
 <h2 id="2-member-conduct">2. Member Conduct</h2>
 <p>You agree not to use the Service to:</p>
-<ul>
+<ol>
     <li>attempt to gain unauthorized access to the Service or to any servers, systems, networks, databases, or other technology or data involved in its provision;</li>
     <li>violate any applicable law or regulation;</li>
     <li>post content which you do not have the right or permission to post, or to infringe any copyright or other proprietary right of any entity or person;</li>
     <li>post content that is hateful, discriminatory, harmful, illegal, or otherwise objectionable;</li>
```

Comment:
Can we add an HTML comment that this point is referenced later by number, i.e. don't make changes in this list that could cause the numbering here to change?